--- src/main/kotlin/net/internetisalie/lunar/lang/insight/hint/LuaTypeInlayHintProvider.kt
+++ src/main/kotlin/net/internetisalie/lunar/lang/insight/hint/LuaTypeInlayHintProvider.kt
@@ -27,6 +27,7 @@
 
                         val types = LuaTypesVisitor.getTypes(element)
                         val type = types.getValueType(element)
+                        println("ELEMENT " + element.text + " TYPE: " + type.displayName())
 
                         if (type != LuaGraphType.Any && type != LuaGraphType.Undefined) {
                             val typeName = type.displayName()
